src/components/views/rooms/RoomDetailList.js

```diff
@@ -138,7 +141,7 @@ export default React.createClass({
                 </tbody>
             </table>;
         }
-        return <div className="mx_RoomDetailList">
+        return <div className={classNames("mx_RoomDetailList", this.props.className)}>
```

Comment:
surprised our horrible linter doesn't whine about the lack of whitespace around the curly braces